src/triggers/missing.ts

```diff
@@ -12,7 +12,7 @@ export const ethereumMissingCreatedAtTime: (tableName: Table) => Trigger<undefin
 };
 
 export const missingMetadataObject: Trigger<undefined> = async (clients) => {
-  const nftQuery = `select * from ${Table.nfts} n
+  const nftQuery = `select * from "${Table.nfts}" n
```

Comment:
so theres 2 cases that could result in issue from this PR change.
1 - as in this example, where we need "" around a table name since its got uppercase now
2 - if there are any hardcoded table names anywhere else in the code that don't refer to the Table object

did you run through things pretty comprehensively to ensure we've covered both 1 and 2 in all files? i'll double check over too

also, this makes me think this PR could also a good opportunity for us to rename to snake_case instead of camelCase for SQL coz camelCase seems consistently annoying in sql lol?

Reply:
yeah I checked for all hardcoded table names and replaced with a reference to Table, but also wrapped with "". And yeah I think changing to snake_case is a good idea that will make this less annoying

Reply:
probably should open another PR renaming field names to snake_case too